### ISSUE TITLE
UnusedVariableSniff false positive

### ISSUE THREAD

**Issue by janlanger** [slevomat/coding-standard#466]
```php
$retryCounter = 0;

while ($retryCounter++ <= self::TRANSACTION_RETRY_LIMIT) {
 //....
}
```

This code reports error Unused variable $retryCounter. (SlevomatCodingStandard.Variables.UnusedVariable.UnusedVariable), but the variable is used.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/88033f575e9c9a81474ff0287ea253f17ddfd8f4
